sktime/transformers/summarise.py

```diff
@@ -215,3 +228,286 @@ def transform(self, X, y=None):
         Xt = pd.DataFrame(Xt)
         Xt.columns = self.columns_
         return Xt
+
+
+class TsFreshFeatureExtractor(BaseTransformer):
+    
+    def __init__(self,default_fc_parameters=None,kind_to_fc_parameters=None,
+                    chunksize=defaults.CHUNKSIZE,
+                     n_jobs=defaults.N_PROCESSES, show_warnings=defaults.SHOW_WARNINGS,
+                     disable_progressbar=defaults.DISABLE_PROGRESSBAR,
+                     impute_function=defaults.IMPUTE_FUNCTION,
+                     profile=defaults.PROFILING,
+                     profiling_filename=defaults.PROFILING_FILENAME,
+                     profiling_sorting=defaults.PROFILING_SORTING,
+                     distributor=None):
+        self.default_fc_parameters = default_fc_parameters
+        self.kind_to_fc_parameters = kind_to_fc_parameters
+        self.n_jobs = n_jobs
+        self.chunksize = chunksize
+        self.show_warnings = show_warnings
+        self.disable_progressbar = disable_progressbar
+        self.impute_function = impute_function
+        self.profile = profile
+        self.profiling_sorting = profiling_sorting
+        self.profiling_filename = profiling_filename
+        self.distributor = distributor
+        self.passed_default_fc_params = None
+        self.passed_kind_to_fc_params = kind_to_fc_parameters
+
+        #time series attribute
+        self.timeseries_container = None
+
+    def set_timeseries_container(self, timeseries_container):
+        self.timeseries_container = timeseries_container    
+
+    def _check_default_rc_parameters(self):
```

Comment:
I think most of this is handled by tsfresh internally, no? I'd simply accept three string values or any tsfresh fc parameter class. For string values, I'd simply look up the corresponding tsfresh parameter class in a dictionary.